README.md

```diff
@@ -81,34 +81,16 @@ geckodriver
 
 #### c) Selenium standalone server
 
-[Selenium server](https://selenium.dev/downloads/) is useful especially when you need to execute multiple tests at once
-or your tests are run in different browsers - like on your CI server.
-
-Selenium server receives commands and starts new sessions using browser drivers acting like hub distributing the commands
-among multiple nodes.
+Selenium server is useful especially when you need to execute multiple tests at once, if your tests are run
+in different browsers (like on your CI server) or eg. when you need to distribute tests amongst multiple machines
```

Comment:
Selenium server can be useful when you need to execute multiple tests at once, when you run tests in several different browsers (like on your CI server), or when you need to distribute tests amongst several machines in grid mode (where one Selenium server acts as a hub, and others connect to it as nodes).